### ISSUE TITLE
Recent update has changed non-target nameplate (classic)

### ISSUE THREAD

**Issue by evind** [kesava-wow/kuinameplates2#463]
Hi, as of the last update my nameplates have changed to look like [this](https://i.imgur.com/ubgnVou.png) for mobs that are not targeted. Targeted mobs look like [this](https://i.imgur.com/IVFV6Zm.png), which is how untargeted mobs used to look minus the background glow. I can't find how to change this in the settings so have reverted to the previous version for now.

**Comment by kesava-wow** [comment]
Uncheck "Hide non-tracked names" in /knp > text, and see the tooltip. The dropdown it mentions doesn't exist on classic but the checkboxes in interface > names still affect it.

**Comment by evind** [comment]
Thanks for your reply. I unchecked "Hide non-tracked names" and my nameplates went back to how they were!

In regards to how this might have happened: I am multi-boxing two chars on two separate accounts, both with kui set up the same. I was playing last night and my nameplates were normal. I updated my addons via the twitch app this morning and logged in when I noticed the change in nameplate appearance. The fact the same change occurred on both accounts and there being an update for kui led me to believe the update might have caused it. I didn't tweak my settings recently and don't have "NPC names" checked for either characters.
